src/orb.yml

```diff
@@ -30,12 +30,16 @@ commands:
         type: integer
         default: 3
         description: "Parameter for `bundle install --retry`"
+      bundle_path:
+        type: string
+        default: vendor/bundle
+        description: "Parameter for `bundle install --path`"
     steps:
       - restore_cache:
           keys: 
             - bundle-cache-{{ checksum "Gemfile.lock" }}
             - bundle-cache
-      - run: bundle install --jobs=<< parameters.jobs >> --retry=<< parameters.retry >> --path vendor/bundle
+      - run: bundle install --jobs=<< parameters.jobs >> --retry=<< parameters.retry >> --path << parameters.bundle_path >>
       - save_cache:
           key: bundle-cache-{{ checksum "Gemfile.lock" }}
           paths:
```

Comment:
This `paths` option should also be parameterized.

Reply:
oops

Reply:
Apply this in bb79737